src/Coduo/PHPHumanizer/DateTime/PreciseFormatter.php

```diff
@@ -38,8 +38,14 @@ public function formatDifference(PreciseDifference $difference, $locale = 'en')
                 $locale
             );
         }
-        $suffix = $difference->isPast() ? 'compound.ago' : 'compound.from_now';
 
-        return implode(', ', $diff).' '.$this->translator->trans($suffix, array(), 'difference', $locale);
+        $translationKey = $difference->isPast() ? 'past' : 'future';
+
+        return $this->translator->trans(
+            'compound.' . $translationKey,
```

Comment:
No need for ``$translationKey``, ``'compound.' . $difference->isPast() ? 'past' : 'future'`` its enoguh